internal/component/database_observability/mysql/component.go

```diff
@@ -29,7 +29,11 @@ import (
 	"github.com/grafana/alloy/syntax/alloytypes"
 )
 
-const name = "database_observability.mysql"
+const (
+	name        = "database_observability.mysql"
+	querySample = "query_sample"
+	schemaTable = "schema_table"
```

Comment:
The collectors have a `Name()` method that returns a similar string. Maybe we could put that into a constant and reuse it from here? (the format is `SchemaTable` rather than `schema_table`, I'm happy with underscores)

Reply:
good idea, thanks!